test/coa/_index.js

```
@@ -0,0 +1,84 @@
+'use strict';
+
+var FS = require('fs'),
+    SVGO = require(process.env.COVERAGE ?
+        '../../lib-cov/svgo/coa.js' :
+        '../../lib/svgo/coa.js').api,
+    PATH = require('path'),
+    svgPath = PATH.resolve(__dirname,'test.svg');
+
+describe('coa', function () {
+    var output;
+
+    beforeEach(function () {
+        output = '';
+    })
+
+    function replaceConsoleLog() {
+        var initialConsoleLog = global.console.log;
+
+        global.console.log = function () {
+            output += arguments[0];
+
+            initialConsoleLog.apply(console, arguments);
+        }
+    }
+
+    function replaceConsoleError() {
+        var initialConsoleError = global.console.error;
+
+        global.console.error = function () {
+            output += arguments[0];
+
+            initialConsoleError.apply(console, arguments);
+        }
+    }
+
+    it('should show plugins', function (done) {
+        replaceConsoleLog();
+
+        SVGO({ 'show-plugins': true })
+            .then(() => { /Currently available plugins:/.test(output) ? done() : null; })
+            .catch();
+    });
+
+    it('should throw error in opts.config when is not parsed string as value', function (done) {
```

`should throw AN error if "config" can not be parsed`